history-network.md

```diff
@@ -209,7 +209,7 @@ encoded_uncles          = rlp.encode(list_of_uncle_headers)
 Note 1: The type-specific encoding might be different in future transaction types, but this encoding
 works for all current transaction types.
 
-Note 2: The `list_of_uncle_headers` refers to the array of uncle headers [defined in the devp2p spec](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#block-encoding-and-validity).  
+Note 2: The `list_of_uncle_headers` refers to the list of uncle headers [defined in the Ethereum yellow paper](https://github.com/ethereum/yellowpaper/blob/30782852fef61f61a247dafbdad814ae7e00fb39/Paper.tex#L399) and is represented as an array of byte arrays where each byte array is the raw bytes representing an uncle header.  
```

Comment:
Yeah, the way @kdeme describes it pretty natural to me -- with the clarification that "an RLP encoded list of block headers (from the uncle blocks)" is the definition of `encoded_uncles`, not `list_of_uncle_headers`.

I'd love to find a version that we all find helpful and informative. Here's another take on the text:
```suggestion
Note 2: The `list_of_uncle_headers` refers to the list of uncle headers [defined in the Ethereum yellow paper](https://github.com/ethereum/yellowpaper/blob/30782852fef61f61a247dafbdad814ae7e00fb39/Paper.tex#L414). As usual, when encoding to RLP, each header must first be formatted as a list where each field is represented as an integer or byte-string. So `list_of_uncle_headers` is a list of these lists, before rlp-encoding.  
```
Part of me thinks we can drop the whole explanation. I wouldn't feel a need to explain `rlp.encode(header)` in a spec, for example. That also requires you to convert each of the fields of the header into an `int` or `bytes`, too.

I can understand the desire to help folks remember how RLP works with lists of objects, which is a little less common, though. Then, it's probably important to also note that some of the fields are represented as integers instead of bytes.

The "As usual" is a nice cue for people who recognize this as standard practice for rlp-encoding lists. Without that, it reads like we're trying to tell folks that some abnormal encoding is happening, instead.

Reply:
Any of these outcomes is fine with me ultimately.  I just know it took me a minute to get what was going on and I didn't recognize the `list_of_uncle_headers` reference back to the yellow paper (though maybe that's more explainable by my not understanding core concepts well enough more than anything).